common/app/routes/challenges/Test-Suite.jsx

```diff
@@ -8,6 +8,24 @@ const propTypes = {
   tests: PropTypes.arrayOf(PropTypes.object)
 };
 
+function getAccessibleText(err, pass, text) {
+  let accessibleText = '';
+
+  // Determine test status (i.e. icon)
+  if (err) {
+    accessibleText += 'Error ';
+  } else if (pass) {
+    accessibleText += 'Pass';
+  } else {
+    accessibleText += 'Waiting';
+  }
+
+  // Append the text itself
+  accessibleText += ' - ';
+  accessibleText += text;
+  return accessibleText;
```

Comment:
These three lines can be condensed into one simple string concatenation.
`return accessibleText + '-' + text;`